Update minimum Node version on README.md

**Describe the issue you encountered:** Update minimum Node version on README.md.

With `v7.3.0` I got an unit test error, which is:

````
  1444 passing (17s)
  47 pending
  1 failing

  1) downloadsBar component very narrow downloads bar with items renders no downloads:

      AssertionError: 3 == 0
      + expected - actual

      -3
      +0
      
      at Context.<anonymous> (test/unit/app/renderer/components/download/downloadBarTest.js:118:14)
````

The error was fixed by updating Node to `v7.9.0`.

We could update it to 8 instead of it, though I am not sure if the browser is fully compatible with that newer version.

- Platform (Win7, 8, 10? macOS? Linux distro?): macOS

- Brave Version (revision SHA): master

- Steps to reproduce:
    1. git push

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

--- a/README.md
+++ b/README.md
@@ -51,13 +51,9 @@ For other platforms (macOS, Linux) You'll need certain packages installed before
 
 ### Prerequisites
 
-1. `nodejs` **`>= 6.1`**
+1. `nodejs` **`>= 7.9.0`**
 
     Install from your package manager or download from https://nodejs.org
-
-2. `node-gyp` **`3.3.1`**
-
-        sudo npm install -g node-gyp@3.3.1
 
 #### On Debian / Ubuntu /Mint
 
@@ -91,11 +87,9 @@ After installing the prerequisites:
 
         cd browser-laptop
 
-3. Install the Node (v5+) dependencies:
+3. Install the Node dependencies:
 
         npm install
-
-If this fails on Linux with an error related to `ad-block`, try updating to Node 6.1 and `node-gyp` 3.3.1 (see discussion at https://github.com/brave/browser-laptop/issues/214)
 
 Instead of `npm install` you may also install with [yarn](https://github.com/yarnpkg/yarn).
 

--- a/package.json
+++ b/package.json
@@ -173,7 +173,7 @@
     "mockery": "^1.7.0",
     "muon-winstaller": "^2.5.4",
     "ncp": "^2.0.0",
-    "node-gyp": "^3.2.1",
+    "node-gyp": "^3.3.1",
     "node-libs-browser": "^1.0.0",
     "node-static": "^0.7.7",
     "nsp": "^2.2.0",
@@ -236,6 +236,6 @@
   },
   "engineStrict": true,
   "engines": {
-    "node": ">= 7.0.0"
+    "node": ">= 7.9.0"
   }
 }

